paying_for_college/templates/worksheet.html

```diff
@@ -2115,7 +2115,7 @@ <h4 class="metric_heading">
                                         </label>
                                       </div>
                                       <p>
-                                        <a href="?">
+                                        <a href="https://studentaid.ed.gov/sa/repay-loans/understand/plans">
```

Comment:
Please add `target="_blank" rel="noopener"` as attributes (since that's the way this tool treats all its external links).